Driverless AI/scoring-pipeline-deployment-to-apache-nifi/scoring-pipeline-deployment-to-apache-nifi.md

```diff
@@ -0,0 +1,352 @@
+- Basic knowledge of Driverless AI or doing the following tutorials:
+    - [Automatic Machine Learning Introduction with Driverless AI Test Drive Tutorial](https://training.h2o.ai/products/tutorial-1a-automatic-machine-learning-introduction-with-driverless-ai)
+    - [Tutorial 4A: Scoring Pipeline Deployment Introduction](https://training.h2o.ai/products/tutorial-4a-scoring-pipeline-deployment-introduction#tab-product_tab_overview)
+    - [Tutorial 4B: Scoring Pipeline Deployment Templates](https://training.h2o.ai/products/tutorial-4b-scoring-pipeline-deployment-templates)
+    - [Tutorial 4C: Scoring Pipeline Deployment in Java Runtime](https://training.h2o.ai/products/tutorial-4c-scoring-pipeline-execution-in-java-runtime)
+- Downloaded the Driverless AI Deployment Repo to your local machine since we will be using the NiFi Data Flow xml templates that come with mojo-nifi/ folder.
+
+```bash
+git clone -b mojo-nifi https://github.com/james94/dai-deployment-examples/
+```
+
+## Task 1: Set Up Environment
+
+### Create an Environment Directory Structure
+
+```bash
+# Create directory structure for DAI MOJO NiFi Projects
+
+# Create directory where the mojo-pipeline/ folder will be stored
+mkdir $HOME/dai-mojo-nifi/
+
+# Create input directory used for batch scoring or real-time scoring
+mkdir -p $HOME/dai-mojo-nifi/testData/{test-batch-data,test-real-time-data}
+```
+
+### Set Up Driverless AI MOJO Requirements
+
+#### Download MOJO Scoring Pipeline
+
+1\. If you have not downloaded the MOJO Scoring Pipeline, go to [Tutorial 4B: Scoring Pipeline Deployment Templates](https://training.h2o.ai/products/tutorial-4b-scoring-pipeline-deployment-templates), then go to **Task 1: Set Up Environment**, then go to section **Download MOJO Scoring Pipeline** to download it. When finished, come back to this tutorial.
+
+2\. Move the **mojo.zip** file to `dai-mojo-nifi/` folder and then extract it:
+
+```bash
+cd $HOME/dai-mojo-nifi/
+mv $HOME/Downloads/mojo.zip .
+unzip mojo.zip
+```
+
+#### Install MOJO2 Java Runtime Dependencies
+
+3\. Download and install Anaconda
+
+```bash
+# Download Anaconda
+wget https://repo.anaconda.com/archive/Anaconda3-2020.02-Linux-x86_64.sh
+
+# Install Anaconda
+bash Anaconda3-2020.02-Linux-x86_64.sh
+```
+
+4\. Create virtual environment and install required packages
+
+```bash
+# Install Python 3.6.10
+conda create -y -n model-deployment python=3.6
+conda activate model-deployment
+# Install Java
+conda install -y -c conda-forge openjdk=8.0.192
+
+# Install Maven
+conda install -y -c conda-forge maven
+```
+
+#### Set Driverless AI License Key
+
+5\. Set the Driverless AI License Key as a temporary environment variable
+
+```bash
+# Set Driverless AI License Key
+export DRIVERLESS_AI_LICENSE_KEY="{license-key}"
+```
+
+### Prepare Hydraulic Test Data For Mojo NiFi Scoring
+
+Make sure there is **input test data** in the input directory NiFi will be pulling data from.
```

Comment:
rewrite this: "Make sure there is **input test data** in the input directory NiFi will be pulling data from."

to this: "Make sure there is **input test data** in the input directory NiFi."